<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="79" started-at="2019-09-16T09:14:59Z" finished-at="2019-09-16T09:15:00Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="79" started-at="2019-09-16T09:14:59Z" finished-at="2019-09-16T09:15:00Z">
      <class name="com.ProjectName.ModuleName.testCases.LoginToApp2">
        <test-method status="FAIL" signature="loadingFiles()[pri:0, instance:com.ProjectName.ModuleName.testCases.LoginToApp2@498d318c]" name="loadingFiles" is-config="true" duration-ms="0" started-at="2019-09-16T14:45:00Z" finished-at="2019-09-16T14:45:00Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at java.io.File.<init>(Unknown Source)
	at com.relevantcodes.extentreports.Report.setFilePath(Report.java:527)
	at com.relevantcodes.extentreports.ExtentReports.<init>(ExtentReports.java:80)
	at com.relevantcodes.extentreports.ExtentReports.<init>(ExtentReports.java:420)
	at com.ProjectName.ModuleName.testBase.TestBase.loadingFiles(TestBase.java:50)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:216)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:590)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loadingFiles -->
        <test-method status="SKIP" signature="login()[pri:0, instance:com.ProjectName.ModuleName.testCases.LoginToApp2@498d318c]" name="login" duration-ms="0" started-at="2019-09-16T14:45:00Z" finished-at="2019-09-16T14:45:00Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="SKIP" signature="closesuite()[pri:0, instance:com.ProjectName.ModuleName.testCases.LoginToApp2@498d318c]" name="closesuite" is-config="true" duration-ms="0" started-at="2019-09-16T14:45:00Z" finished-at="2019-09-16T14:45:00Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closesuite -->
      </class> <!-- com.ProjectName.ModuleName.testCases.LoginToApp2 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
